packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -405,13 +405,16 @@ contract Deploy is Deployer {
         deployImplementations();
         initializeImplementations();
 
-        setAlphabetFaultGameImplementation({ _allowUpgrade: false });
```

Comment:
Why is this normally deployed on non-test chains?

Reply:
OP Mainnet and OP Sepolia does not have deployed the Alphabet, Asterisc, and Fast games. My understanding is that they have not used the `Deploy.s.sol` for deploying those networks, and this  is more intended for their tests.

```
╰─ cast call -r https://eth.llamarpc.com 0xe5965Ab5962eDc7477C8520243A95517CD252fA9 "gameImpls(uint32) (address)" 0
0xA6f3DFdbf4855a43c529bc42EDE96797252879af

╰─ cast call -r https://eth.llamarpc.com 0xe5965Ab5962eDc7477C8520243A95517CD252fA9 "gameImpls(uint32) (address)" 1
0x050ed6F6273c7D836a111E42153BC00D0380b87d

╰─ cast call -r https://eth.llamarpc.com 0xe5965Ab5962eDc7477C8520243A95517CD252fA9 "gameImpls(uint32) (address)" 2
0x0000000000000000000000000000000000000000

╰─ cast call -r https://eth.llamarpc.com 0xe5965Ab5962eDc7477C8520243A95517CD252fA9 "gameImpls(uint32) (address)" 254
0x0000000000000000000000000000000000000000

╰─ cast call -r https://eth.llamarpc.com 0xe5965Ab5962eDc7477C8520243A95517CD252fA9 "gameImpls(uint32) (address)" 255
0x0000000000000000000000000000000000000000
```

```
╰─ cast call -r https://ethereum-sepolia-rpc.publicnode.com 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1 "gameImpls(uint32) (address)" 0
0x5e0877a8F6692eD470013e651c4357d0C4941e6C

╰─ cast call -r https://ethereum-sepolia-rpc.publicnode.com 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1 "gameImpls(uint32) (address)" 1
0x4Ed046e66c96600DaE1a4ec39267bB0cE476E8cc

╰─ cast call -r https://ethereum-sepolia-rpc.publicnode.com 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1 "gameImpls(uint32) (address)" 2
0x0000000000000000000000000000000000000000

╰─ cast call -r https://ethereum-sepolia-rpc.publicnode.com 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1 "gameImpls(uint32) (address)" 254
0x0000000000000000000000000000000000000000

╰─ cast call -r https://ethereum-sepolia-rpc.publicnode.com 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1 "gameImpls(uint32) (address)" 255
0x0000000000000000000000000000000000000000
```